lib/SILOptimizer/FunctionSignatureTransforms/ExistentialSpecializer.cpp

```diff
@@ -101,7 +101,8 @@ static bool findConcreteType(ApplySite AI, int ArgIdx, CanType &ConcreteType) {
     SILValue InitExistential =
         findInitExistentialFromGlobalAddrAndApply(GAI, AI, ArgIdx);
     /// If the Arg is already init_existential, return the concrete type.
-    if (findConcreteTypeFromInitExistential(InitExistential, ConcreteType)) {
+    if (InitExistential &&
```

Comment:
This fixes the  case where global_addr analysis can not find an init_existential. See "global_addr_init" test case below.